<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="txtUserName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="labAddUsers">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color:qlineargradient(spread:pad, x1:0.506975, y1:0.432, x2:0.99, y2:0.437227, stop:0.900498 rgba(4, 0, 62, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(255, 255, 255);
 font: 75 15pt &quot;Motken K Sina&quot;;
</string>
   </property>
   <property name="text">
    <string>اضافة مستخدم جديد</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 75 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0);
background-color:none;</string>
   </property>
   <property name="text">
    <string>إسم المستخدم</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 75 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0);
background-color:none;</string>
   </property>
   <property name="text">
    <string>الاسم : </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 75 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0);
background-color:none;</string>
   </property>
   <property name="text">
    <string>كلمة المرور :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboAccountType">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	
	border: 1px solid #5b586e;

</string>
   </property>
   <item>
    <property name="text">
     <string>مسؤول</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>مستخدم عادي</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 75 10pt &quot;Motken K Sina&quot;;
color:rgb(0, 0, 0);
background-color:none;</string>
   </property>
   <property name="text">
    <string>نوع الحساب</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveUser">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>103</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>حفــظ</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/true.png</normaloff>:/icons_qrc/true.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingUser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>103</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Motken K Sina</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>الغــــاء</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/exclamation-48.ico</normaloff>:/icons_qrc/exclamation-48.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTeacherName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	
	border: 1px solid #5b586e;

</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboAccountType</tabstop>
  <tabstop>txtUserName</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>labAddUsers</tabstop>
 </tabstops>
 <resources>
  <include location="icons_rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
